connect.markdown

```diff
@@ -0,0 +1,23 @@
+---
+layout: fullwidth
+primary_title: Connect
+title: Connect with the Community
+---
+
+## Connect with the Community
+
+---
+
+There are several ways to connect with the OpenSearch community.
+
+* OpenSearch has [active forums](https://discuss.opendistrocommunity.dev/), which are the fastest way to get help using, troubleshooting, and developing with OpenSearch. As a bonus, questions and discussions on the forums are easy for others to learn from in the future. 
+* If you want to share a feature idea or submit a bug report, open an issue in the relevant repo in [the GitHub project](https://github.com/opensearch-project). 
+* If you want to present or have a question for an upcoming [community meeting](https://opensearch.org/events/), add a comment to the agenda for that meeting.
+* Businesses partnering in the development and advancement of OpenSearch can propose to [add themselves](https://github.com/opensearch-project/project-website#adding-to-the-partners-page) to the [partners](https://opensearch.org/partners/) page with a pull request.
+* To feature a plug-in or project you developed with OpenSearch, add it to the [community projects](https://opensearch.org/community_projects) page with a pull request.
+* Information about using the OpenSearch brand and logo are available on the [Brand Guidelines](https://opensearch.org/brand.html) page.
+* To report or inquire about a potential security vulnerability, see the [Reporting a Vulnerability guidelines](https://github.com/opensearch-project/.github/blob/main/SECURITY.md).
```

Comment:
@stockholmux aside from bolding, what could be done to make the security point more prominent. Moving it to the top?